src/lib/input/input.scss

```diff
@@ -121,3 +121,21 @@ textarea.mat-input-element {
   padding: 2px 0;
   margin: -2px 0;
 }
+
+// Remove select button from IE11
+select::-ms-expand {
+  display: none;
+}
```

Comment:
This will target _all_ `<select>` elements; it should be restricted to a CSS class that's applied by the directive